static/js/controllers/inbox.js

```diff
@@ -148,6 +148,16 @@ require('moment/locales');
         $scope.items = contacts || [];
       };
 
+      $scope.removeContact = function(contact) {
```

Comment:
N.B. I made the renames on `develop` and then rebased this PR onto latest `develop`